pydough/errors/error_utils.py

```diff
@@ -167,28 +168,122 @@ class ValidSQLName(PyDoughPredicate):
     as the name for a SQL table path/column name.
     """
 
-    # Regex for unquoted SQL identifiers
-    _UNQUOTED_SQL_IDENTIFIER = re.compile(
-        r"^[A-Za-z_][A-Za-z0-9_]*(\.[A-Za-z_][A-Za-z0-9_]*)*$"
-    )
+    # Single-part unquoted SQL identifier (no dots here).
+    _UNQUOTED_SQL_IDENTIFIER = re.compile(r"^[A-Za-z_][A-Za-z0-9_]*$")
```

Comment:
Don't prefix this one with `_`, and use a multiline string docstring beneath this kind of class variable, so that it will show up in tooltips

```suggestion
    UNQUOTED_SQL_IDENTIFIER = re.compile(r"^[A-Za-z_][A-Za-z0-9_]*$")
    """
    Regex pattern for a single-part unquoted SQL identifier (without dots).
    """
```

Reply:
Done!